src/styles/index.scss

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  &__container {
+    width: 80vmin;
+    height: 80vmin;
+    border: dotted 1vmin #000;
+    border-radius: 50%;
+    position: relative;
+  }
+
+  &__minute,
+  &__seconds,
+  &__center {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    transform-origin: bottom center;
+  }
+
+  &__minute {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    animation: rotate-minute 3600s infinite linear;
+  }
+
+  &__seconds {
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    animation: rotate-second 60s infinite linear;
+  }
+
+  &__center {
+    width: 5vmin;
+    height: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+    z-index: 1;
+  }
+
+  @keyframes rotate-minute {
+    from {
+      transform: translate(-50%, -100%) rotate(0deg);
+    }
+    to {
+      transform: translate(-50%, -100%) rotate(360deg);
+    }
+  }
+
+  @keyframes rotate-second {
+    from {
+      transform: translate(-50%, -100%) rotate(0deg);
+    }
+    to {
+      transform: translate(-50%, -100%) rotate(360deg);
+    }
+  }
+
+  /* Модифікатор speed-up застосовується до блоку stopwatch */
+  &--speed-up {
+    &__seconds {
+      animation-duration: 10s; /* Швидкість для секундної стрілки */
+    }
+
+    &__minute {
+      animation-duration: 600s; /* Швидкість для хвилинної стрілки */
```

Comment:
The `animation-duration` for `.stopwatch--speed-up &__minute` also changes the speed of the minute hand. Verify that the duration of 600s is correct according to the requirements. If the speed-up is not intentional or does not match the desired effect, it should be corrected.